### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by manojn97**
# Pull Request Review

## Checklist

- [x] Issue: 3394
- [x] List of related PRs. : None
- [x] Project builds without any errors.
- [x] Unit Tests updated (or created where needed) and all pass.
- [x] You have ran manual functional “smoke” test (does product as a whole still work?).
- [x] User Story meets the acceptance criteria - and passes. acceptance criteria should be understood at outset.
- [x] There is a PR for update of internal documentation for the affected Project(s) - README etc.
- [x]  **Description of the changes**: Vulnerabilities have been reduced after making the code changes by removing the  helm2 version in the Dockerfile. Vulnerabilities that have been removed can be found in 3394

### CODE REVIEW

**Review by hhellyer — CHANGES_REQUESTED** [review]
Searching this repository for the removed helm version `2.16.9` finds various references in code and documentation. It looks like that is still the default to use even after removal. Those references need investigating, resolving and testing before this PR can be merged. The docs in particular will need to be updated with the new default version and that there is now only helm 3 available.
Also there is no reference in the PR or issue as to why helm 2 is being removed which will not help future maintainers of this project understand the changes.

**Comment by manojn97** [comment]
will open a new PR after making changes based on https://github.ibm.com/TNC/tnc-o-tracking/issues/3473

**Pull request closed without merging**
